examples/utils/utils.go

```diff
@@ -533,14 +570,55 @@ func mapClusterToKubeAccess(kubeconfigs ...interface{}) (clusterKubeAccessMap, e
 		}
 
 		// Parse the kubeconfig user auth exec args for the cluster name.
-		clusterNameIndex := len(kubeAccess.RESTConfig.ExecProvider.Args) - 1
+		var clusterNameIndex int
```

Comment:
This previously assumed that the cluster name is always the last parameter in the list, but that's not true (with the current aws cli version it's the output type for example).

This now retrieves the correct positional argument